src/update-promoted-values.ts

```diff
@@ -150,3 +250,37 @@ function isCollectionPath(value: unknown): value is CollectionPath {
 function isCollectionIndex(value: unknown): value is CollectionIndex {
   return typeof value === 'string' || typeof value === 'number';
 }
+
+async function getRelevantCommits(
+  prevTag: string,
+  nextTag: string,
+  dockerImageRepository: string,
+  repoURL: string,
+  gitHubClient: GitHubClient,
+  dockerRegistryClient: DockerRegistryClient,
+): Promise<RelevantCommit[]> {
+  const commits = await dockerRegistryClient.getGitCommitsBetweenTags({
+    prevTag,
+    nextTag,
+    dockerImageRepository,
+  });
+
+  if (commits.length <= 0) return [];
```

Comment:
Do we need to add handling for the case there is 1 commit, or will the github api handle that okay?

Reply:
Good call out! I tested locally and adding `^` to the head commit made it work locally. Will pay attention to that in the actual PRs to see what comes out -- I expect most PRs will be created with a single commit?